cogs/clash_river_war.py

```diff
@@ -245,15 +279,20 @@ async def clanRemainingWarAttacks(self, ctx, clan_tag: str):
             all_clan_attacks = self.clash_royale_service.clan_remaining_war_attacks(clan_tag)
         embed = remaining_war_embed(all_clan_attacks)
         await ctx.send(embed=embed)
-    @riverwar.error
-    async def riverwar_error(self, ctx, error):
-        if isinstance(error, commands.MissingRequiredArgument):
-            await ctx.send(embed=discord.Embed(
-                description="Incorrect arguments entered.\nCorrect usage: **!riverwar [clan_tag] [past_weeks]**" +
-                "\nOr use the AUSCLAN version: **!ausclanwar**",
-                colour=discord.Colour.red()
-            ))
 
+    @commands.command(name="ausclanwarplayers", pass_context=True)
+    async def ausclanWarPlayers(self, ctx):
+        async with ctx.typing():
+            all_current_war_players = self.clash_royale_service.clan_players_remaining_war_attacks('9GULPJ9L')

```

Comment:
Yea, I think a better name for function etc is `clan_players_unfinished_war_attacks`

Reply:
done